### ISSUE TITLE
Unique constraint failure

### ISSUE THREAD

**Issue by frehulfd** [johnspurlock/app-privacy-report-viewer#5]
Dragged my JSON file in and got this error:

```
SqliteError: UNIQUE constraint failed: domain3.filename, domain3.bundleId, domain3.domain, domain3.context, domain3.initiatedType
```

[App_Privacy_Report_v4_2021-09-22T10_41_45.json.zip](https://github.com/johnspurlock/app-privacy-report-viewer/files/7211691/App_Privacy_Report_v4_2021-09-22T10_41_45.json.zip)

**Comment by johnspurlock** [comment]
Thanks, taking a look

**Comment by johnspurlock** [comment]
Great test case, updated the underlying data model that should fix this, and released a new version: 0.7

`deno run --allow-net --allow-env --allow-read=.,`which deno` --allow-write=. https://raw.githubusercontent.com/johnspurlock/app-privacy-report-viewer/v0.7/src/app.ts`

**Comment by frehulfd** [comment]
Awesome, thanks so much!
